README.md

```diff
@@ -130,6 +154,20 @@ Following are example declarative and programmatic configuration snippets.
 </hazelcast-client>
 ```
 
+#### YAML Configuration
+
+```yml
+hazelcast:
+  network:
```

Comment:
`join` key is missing after network.

Reply:
Are you sure? I didn't see in XML above

Reply:
sorry my bad. Just Ignore. 
I missed that it is client config. 
https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/resources/hazelcast-client-full-example.yaml#L237